apps/masterbots.ai/types/types.ts

```
@@ -5,6 +5,7 @@ import 'next-auth'
 import type { DefaultSession, DefaultUser } from 'next-auth'
 import type { ChatCompletionMessageParam } from 'openai/resources'
 import type Stripe from 'stripe'
+import React from 'react';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)